documentation/source/development/reference/octree-collision.rst

```diff
@@ -0,0 +1,167 @@
+Multi octree collision detection
+================================
+
+Octree collision detection allows us to find intersections between octree geometry and a ray, for example the camera view ray. This is an essential part for the octree editor. Inexor engine allows to have multiple octrees with arbitrary position and relative size (no support for rotations yet), making collision detection significantly more complex than single octree traversal:
+
+.. image:: octree_collision_sizes.jpg
+    :width: 500
+    :alt: Multiple octree worlds with different relative size.
+
+In the following screenshot, you can see three octrees of different types and different sizes. One octree is completely filled (we call this an ``Cube::Type::OCTANT`` in the engine). The one in the right has empty sub-cubes in it. You can see that these cubes are not indented at all, because indentation is not taken into account yet for octree collision. [#f1]_
```

Comment:
I changed it to this:

```
In the following screenshot, you can see three octrees of different types and different sizes. One octree is filled with 8 sub-cubes (we call a cube which has children``Cube::Type::OCTANT`` in the engine). The one in the right has some empty and some solid sub-cubes in it. You can see that these cubes are not indented at all, because indentation is not taken into account yet for octree collision.
```

Reply:
Done.